src/main/java/mate/academy/hibernate/relations/service/impl/MovieServiceImpl.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import java.time.DateTimeException;
```

Comment:
The import 'java.time.DateTimeException' is not appropriate for this situation. Consider using a custom exception or a more appropriate built-in exception such as 'java.util.NoSuchElementException'.